doc/python/icicle-charts.md

```diff
@@ -305,7 +305,7 @@ def build_hierarchical_dataframe(df, levels, value_column, color_columns=None):
     Levels are given starting from the bottom to the top of the hierarchy,
     ie the last level corresponds to the root.
     """
-    df_all_trees = pd.DataFrame(columns=['id', 'parent', 'value', 'color'])
+    df_trees = []
```

Comment:
I don't understand this change but I'm willing to take it on faith :-)

Reply:
@LiamConnors My only suggestion here (and at the other spot) is that the `df_trees` variable be renamed to clarify that it's a _list_ rather than a df.

Reply:
Yeah, sorry, I should have given more context. 

The original example uses `df.append`, to append each new dataframe in the loop to the first dataframe, but that method was removed. 

So here I instead use a list, and add each to a list and use `pd.concat` on the list instead.

We had to make this update also in the `plotly.express` library itself at one stage and that's the approach we took: https://github.com/plotly/plotly.py/pull/4190

@emilykl good suggestion. Will update the variable name
`pd.concat` has existed at least since https://pandas.pydata.org/pandas-docs/version/0.13/generated/pandas.concat.html